osu.Game.Rulesets.Taiko/Objects/Drawables/Pieces/ElongatedCirclePiece.cs

```diff
@@ -12,18 +14,15 @@ public ElongatedCirclePiece()
             RelativeSizeAxes = Axes.Y;
         }
 
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours)
+        {
+            AccentColour = colours.YellowDark;
+        }
+
         protected override void Update()
         {
             base.Update();
-
-            var padding = Content.DrawHeight * Content.Width / 2;
-
-            Content.Padding = new MarginPadding
-            {
-                Left = padding,
-                Right = padding,
-            };
```

Comment:
Can confirm reproduction of the misalignment; it shows up even on default lazer skin in this PR (or, to be more precise, on f36477e3 with a clean tree), doesn't show up on `master`. Will be visible in `TestSceneAutoplay`. Screenshots:

| `master` | this PR |
| :-: | :-: |
| ![image](https://user-images.githubusercontent.com/20418176/79494942-21a71f00-8024-11ea-892a-905eea43aa3a.png) | ![image](https://user-images.githubusercontent.com/20418176/79494954-28ce2d00-8024-11ea-97e5-0ae68bcd252e.png) |

Seems to be related to removing the padding as highlighted. Bringing it back fixes the alignment of `ElongatedCirclePiece` itself but ticks remain broken (too stretched out).

Reply:
The padding needs a local comment explaining exactly what it's doing until we fix this in a better way. I thought I was able to remove it due to the refactors I made and there was nothing noticeably failing as a result.

Reply:
@bdach can you explain how the offset is occurring? in your screenshots you show it changing at the `DrawableDrumRoll` level, but I can't seem to find any changes I've made which should affect that.

Reply:
Not currently in a position to investigate in detail but from what I remember the actual drumroll's position hasn't changed from `master`, it's the `ElongatedCirclePiece` lower down that starts protruding out of it, breaking alignment.

Reply:
I think I've fixed this by applying at a more central level. Can you check on the solution?

Reply:
Give me a few hours and I'll get back to you on that.

Reply:
Took me a little longer than I promised, but as far as I can see the alignment is fixed, although this set:

https://github.com/ppy/osu/blob/58a1c6e17186084345feedbd1c5b0d44c2ce8695/osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableDrumRoll.cs#L145

doesn't seem to have any effect as the getter is hard-overridden in `DrawableTaikoHitObject` itself:

https://github.com/ppy/osu/blob/8b83c3daabf83075ce0be0c47948a312529c9e1d/osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableTaikoHitObject.cs#L113

so the value gets set but never actually read (draw visualiser shows the one from base).